recipes/coremltools/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
+set -x
+mkdir build
+cd build
+
+cmake \
+    -DCMAKE_OSX_DEPLOYMENT_TARGET=10.13 \
+    -DPYTHON_EXECUTABLE:FILEPATH=${PREFIX}/bin/python \
+    -DPYTHON_INCLUDE_DIR=${PREFIX}/include/python${PY_VER}m/ \
```

Comment:
I think the linux64 build failure at https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=225770&view=logs&j=10bda42a-d420-5c28-a43f-bc4e66405e5c&t=bef775d0-a1a2-55a3-9721-17147029baf0 is is from this.

```
/home/conda/staged-recipes/build_artifacts/coremltools_1602772503997/work/deps/pybind11/include/pybind11/common.h:72:10: fatal error: Python.h: No such file or directory
 #include <Python.h>
          ^~~~~~~~~~
```

Does conda or conda-forge have a standard way of getting the this path (i.e. whether it's `3.8` vs `3.8m`)?

Reply:
This should get you the correct path, not sure whether there is a simpler solution. Normally passing one of `PYTHON_EXECUTABLE`,  `Python_EXECUTABLE`, `Python3_EXECUTABLE` is enough. Note that you may need a different casing based on the CMake version.
```suggestion
    -DPYTHON_INCLUDE_DIR=$(${PYTHON} -c 'import sysconfig; print(sysconfig.get_paths()["include"])') \
```